scripts/5min/Load_parquet_dataframe.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
```

Comment:
general comment could they be moved into a folder called `examples`. This can be under `5min` if you want